<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CS-GAGANDEEP-WS" name="test1" tests="1" failures="1" timestamp="1 Jun 2016 05:40:36 GMT" time="0.032" errors="0">
  <testcase name="delCustomer" time="0.016" classname="com.cloudsmartz.test.TestCustomerAPI">
    <failure type="java.lang.AssertionError" message="expected [&amp;lt;406 Not Acceptable,{}&amp;gt;] but found [&amp;lt;201 Created,{}&amp;gt;]">
      <![CDATA[java.lang.AssertionError: expected [<406 Not Acceptable,{}>] but found [<201 Created,{}>]
	at com.cloudsmartz.test.TestCustomerAPI.delCustomer(TestCustomerAPI.java:53)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- delCustomer -->
</testsuite> <!-- test1 -->
